webdriver_dotnet/SemanticLocators/SemanticLocators.Tests/TestAssignment.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
```

Comment:
Could you add a copyright notice to the source files? Example below. Note this doesn't affect your rights, they're still covered by the CLA.

If you prefer you could use `Copyright 2021 The Semantic Locators Authors` and I'll add an `AUTHORS` FILE

```
//-----------------------------------------------------------------------
// <copyright file="FILENAME.cs" company="Google LLC">
//
// Copyright 2021 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
// http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//
// </copyright>
//-----------------------------------------------------------------------
```

Reply:
Added Copyright 2021 The Semantic Locators Authors in copyright snippets